llvm/lib/Target/AIE/Utils/AIELoopUtils.cpp

```diff
@@ -46,4 +46,14 @@ std::optional<int64_t> getMinTripCount(const MDNode *LoopID) {
   return std::nullopt;
 }
 
+SmallVector<const MachineBasicBlock *, 4>
+getLoopBBs(const MachineLoopInfo &MLI) {
```

Comment:
I'd rename to `getInnermostLoopBlocks`, because you ignore outer loops

Reply:
fixed it